### DISCUSSION TITLE
No way to queue up downloads?

### DISCUSSION THREAD

**Discussion by Booms1981** [JunkFood02/Seal#361] [General]
Hi

Just started using Seal for Android and am really impressed.

One problem seems to be that there isn't an available download queue. If Seal is downloading a YouTube video and I try to line up the next video e.g. by sharing from YouTube to Seal, I get a message saying "an existing download task is already running".

I'd like to be able to line up lots of videos for download and let Seal work through them

**Comment by Marco-9456** [comment]
> Hi
> 
> Just started using Seal for Android and am really impressed.
> 
> One problem seems to be that there isn't an available download queue. If Seal is downloading a YouTube video and I try to line up the next video e.g. by sharing from YouTube to Seal, I get a message saying "an existing download task is already running".
> 
> I'd like to be able to line up lots of videos for download and let Seal work through them

It's planned, See #324 
![Screenshot_20221123_184342_Chrome](https://user-images.githubusercontent.com/41828058/203614450-922f6145-db5b-4a7e-8e9e-91593e495d8e.jpg)
